examples/flutter_gallery/lib/demo/playground/configuration/material_helpers.dart

```diff
@@ -0,0 +1,297 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import '../configuration/property_column.dart';
+
+const double _pickerSelectedElevation = 3.0;
+const double _pickerRowHeight = 46.0;
+
+typedef IndexedValueCallback<T> = Function(int index, T value);
+
+class BorderChoice {
+  BorderChoice({@required this.type, @required this.code});
+  String type;
+  String code;
+}
+
+class ColorChoice {
+  ColorChoice({@required this.color, @required this.code});
+  Color color;
+  String code;
+}
+
+class IconChoice {
+  IconChoice({@required this.icon, @required this.code});
+  IconData icon;
+  String code;
+}
+
+final List<BorderChoice> borderChoices = <BorderChoice>[
```

Comment:
`const List<BorderChoice> borderChoices = ...` 

Likewise for colorChocies, iconChocies